.github/workflows/run-script-and-commit.yml

```
@@ -19,6 +19,13 @@ on:
       user_name:
         required: true
         type: string
+    secrets:
+      remote:
+        required: false
+      GPG_PRIVATE_KEY:
+        required: true
+      PASSPHRASE:
+        required: true
```

nit: to preserve consistency with all secret inputs
```suggestion
    secrets:
      remote:
        required: false
      gpg_private_key:
        required: true
      passphrase:
        required: true
```

---

Done!